src/plotting/plot.component.tsx

```diff
@@ -8,39 +8,39 @@ import {
   VictoryTheme,
   VictoryLegend,
   VictoryTooltip,
+  VictoryGroup,
 } from 'victory';
-import { AxisSettings, PlotType } from '../app.types';
+import {
+  XAxisSettings,
+  PlotDataset,
+  PlotType,
+  YAxisSettings,
+} from '../app.types';
 import { format } from 'date-fns';
 
-export const formatTooltipLabel = (label: number | Date): number | string => {
-  if (label instanceof Date) {
+export const formatTooltipLabel = (
+  label: number,
+  scale: XAxisSettings['scale']
+): number | string => {
+  if (scale === 'time') {
     return format(label, 'yyyy-MM-dd HH:mm:ss');
   }
   return label;
 };
```

Comment:
I've gone back to just using numbers only for all labels, including timestamps, hence why you no longer see 
```
{ [channel: string]: number | Date }
```
This just makes it much easier to manage the sorting and ordering of data. This function is called when viewing a VictoryLabel and when converting unix timestamps for export to CSV, which are the only instances unix timestamps turn into human-readable form

Reply:
No problem, I just thought if we were converting from a `Date` to a unix timestamp and then back to a `Date`again it might be easier to just keep it as `Date`, but yeah it does complicate the types a little. The other thing was it meant I didn't have to match the timestamp column name to it's data in the select function (as you could infer timestamp from the type of `Date`), but you've fixed that :)